[2025-03-10T18:41:47.116+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:41:47.128+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:41:47.128+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-10T18:41:47.147+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): staging_to_curated> on 2025-03-10 12:00:00+00:00
[2025-03-10T18:41:47.162+0000] {standard_task_runner.py:57} INFO - Started process 271 to run task
[2025-03-10T18:41:47.165+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'staging_to_curated', 'scheduled__2025-03-10T12:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmp4fhm42cn']
[2025-03-10T18:41:47.170+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask staging_to_curated
[2025-03-10T18:41:47.194+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T18:41:47.238+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [running]> on host d8442e641ab0
[2025-03-10T18:41:47.288+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='staging_to_curated' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T12:00:00+00:00'
[2025-03-10T18:41:49.831+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-10T18:41:49.856+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=staging_to_curated, execution_date=20250310T120000, start_date=20250310T184147, end_date=20250310T184149
[2025-03-10T18:41:49.890+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-10T18:41:49.920+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10T18:43:41.341+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:43:41.346+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:43:41.346+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-10T18:43:41.352+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): staging_to_curated> on 2025-03-10 12:00:00+00:00
[2025-03-10T18:43:41.358+0000] {standard_task_runner.py:57} INFO - Started process 577 to run task
[2025-03-10T18:43:41.361+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'staging_to_curated', 'scheduled__2025-03-10T12:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmpiub2hyit']
[2025-03-10T18:43:41.368+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask staging_to_curated
[2025-03-10T18:43:41.404+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T18:43:41.438+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [running]> on host d8442e641ab0
[2025-03-10T18:43:41.495+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='staging_to_curated' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T12:00:00+00:00'
[2025-03-10T18:43:43.135+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-10T18:43:43.151+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=staging_to_curated, execution_date=20250310T120000, start_date=20250310T184341, end_date=20250310T184343
[2025-03-10T18:43:43.203+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-10T18:43:43.227+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10T18:50:00.082+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:50:00.090+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [queued]>
[2025-03-10T18:50:00.091+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-10T18:50:00.102+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): staging_to_curated> on 2025-03-10 12:00:00+00:00
[2025-03-10T18:50:00.118+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'staging_to_curated', 'scheduled__2025-03-10T12:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmp6638vh2o']
[2025-03-10T18:50:00.112+0000] {standard_task_runner.py:57} INFO - Started process 271 to run task
[2025-03-10T18:50:00.121+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask staging_to_curated
[2025-03-10T18:50:00.142+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T18:50:00.202+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-10T12:00:00+00:00 [running]> on host 5222d87982cb
[2025-03-10T18:50:00.256+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='staging_to_curated' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T12:00:00+00:00'
[2025-03-10T18:50:03.773+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-10T18:50:03.799+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=staging_to_curated, execution_date=20250310T120000, start_date=20250310T185000, end_date=20250310T185003
[2025-03-10T18:50:03.835+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-10T18:50:03.863+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
